Packs/Logzio/ReleaseNotes/1_1_8.md

```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### Logz.io
+
+- Added functionality to filter Logz.io events by tags.
```

Comment:
```suggestion
- Added the *tags* parameter to support filtering of fetched incidents.
```

Reply:
Wow, I am not sure what happened here. I had all this in there, and I ran a couple commands from demisto-sdk, so I really don't know what happened here. I am fixing.

Reply:
These fixes have been made.